### PR TITLE
feat: account to portfolio exclude asset ids

### PR DESCRIPTION
**Pull request by chxash**
## Description

Fix for https://github.com/shapeshift/web/issues/750

## Notice

Before submitting a pull request, please make sure you have answered the following:

- [X] Have you followed the guidelines in our [Contributing]('https://github.com/shapeshift/web/CONTRIBUTING.md) guide?
- [X] Have you checked to ensure there aren't other open [Pull Requests](https://github.com/shapeshift/web/pulls) for the same update/change?
- [X] Do all new and existing tests pass? Does the linter pass?

## Pull Request Type

- [ ] :bug: Bug fix (Non-breaking Change: Fixes an issue)
- [ ] :hammer_and_wrench: Chore (Non-breaking Change: Doc updates, pkg upgrades, typos, etc..)
- [X] :nail_care: New Feature (Breaking/Non-breaking Change)

## Issue (if applicable)

If applicable, please link to the github issue and put `closes #XXXX` in your comment to auto-close the issue that your PR fixes.

## Testing

Please outline all testing steps

1. Created relevant unit test first
2. After coding, verified that unit test is green
3. Blind tested by hard coding different values for assetIds array, verified the filtering is working fine

### CODE REVIEW

**Comment by cjthompson** [comment]
The `portfolioSlice` tests are currently being refactored so I'd prefer if you waited to merge this until after that refactor and then you can follow the new pattern for your tests.

**Comment by chxash** [comment]
> The `portfolioSlice` tests are currently being refactored so I'd prefer if you waited to merge this until after that refactor and then you can follow the new pattern for your tests.

sure ! will update this PR once the refactoring is merged

**Comment by cjthompson** [comment]
The updated tests have been merged so this PR can be updated against latest develop.

**Comment by chxash** [comment]
> The updated tests have been merged so this PR can be updated against latest develop.

Updated pull request with latest develop

File: src/state/slices/portfolioSlice/utils.test.ts
Comment (DaoDev44) [inline src/state/slices/portfolioSlice/utils.test.ts:39]: Instead of defining the eth and btc accounts at the top level, you can use the `mockETHandBTCAccounts` helper inside `accounts.ts` if you need to change or modify any of the values of an account, the helper function accepts an object for each account. 
```
      const { ethAccount, ethAccount2, ethAccountId, ethAccount2Id } = mockETHandBTCAccounts({
        ethAccountObj: { balance: '1000000000000000000' },
        ethAccount2Obj: { balance: '200000000000000000' }
      })
```

File: src/state/slices/portfolioSlice/utils.test.ts
Comment (DaoDev44) [inline src/state/slices/portfolioSlice/utils.test.ts:99]: After our test refactor, we're moving away from building portfolio object literals and using snapshots for the expected values. You can look in `portolioSlice.test.ts` for reference.

File: src/state/slices/portfolioSlice/utils.test.ts
Comment (DaoDev44) [inline src/state/slices/portfolioSlice/utils.test.ts:321]: Probably want to use snapshot tests for these tests as stated above.

**Review by DaoDev44 — COMMENTED** [review]
Just some general comments on testing practices we're trying to implement post refactor. Thanks for the PR!

**Comment by chxash** [comment]
> Just some general comments on testing practices we're trying to implement post refactor. Thanks for the PR!

Thank you for the feedback, will resolve each comment

File: src/state/slices/portfolioSlice/utils.test.ts
Comment (chxash) [reply src/state/slices/portfolioSlice/utils.test.ts]: Done ! see my updated tests in `portfolioSlice.test.ts`. I moved the tests there as it made more sense (did not understand how snapshot worked at first look).

File: src/state/slices/portfolioSlice/utils.test.ts
Comment (chxash) [reply src/state/slices/portfolioSlice/utils.test.ts]: Done ! Moved my tests from `utils.test.ts` to `portfolioSlice.test.ts`

File: src/state/slices/portfolioSlice/utils.test.ts
Comment (chxash) [reply src/state/slices/portfolioSlice/utils.test.ts]: Done

**Review by DaoDev44 — APPROVED** [review]
LGTM

**Review by 0xdef1cafe — APPROVED** [review]

**Pull request merged**
